lib/manager/reducers/projects.js

```diff
@@ -346,7 +346,11 @@ const projects = (state: ProjectsState = defaultState, action: Action): Projects
       })
     }
     case 'RECEIVE_DEPLOYMENTS': {
-      const {deployments, feedSourceId, projectId} = action.payload
+      const {deployments: deploymentsSummaries, feedSourceId, projectId} = action.payload
+      // Deployment summaries don't include a project id, so we need to add one
+      // Deployment summaries don't include feedversions array, so we create a fake array that gets replaced later
+      const deployments = deploymentsSummaries.map(d => ({...d, projectId, feedVersions: Array(d.numberOfFeedVersions)}))
```

Comment:
Using an array of `undefined` causes a crash when the user navigates from the Deployments tab back to the Feed Sources tab using the browser's Back button (clicking the Feed Sources tab still works).

The error message is as follows:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'feedSource')
    at version.js:176:28
    at Array.find (<anonymous>)
    at getVersionValidationSummaryByFilterStrategy (version.js:175:67)
    ...
```

Reply:
I've fixed this bug, but there is a problem! Since the new "compressed" data doesn't include feed source ids, it's not possible to populate the deployed version column! 

Is there something I'm missing?

Reply:
Ok so then not something I'm missing then. Do we want to include that though? The list of feeds in each deployment can get quite large especially if there are a lot of deployments with lots of feeds.

I'll schedule a time for us to figure this out offline.